index.html

```diff
@@ -50,7 +50,7 @@ <h2>
       <section class="hero-container" id="hero-section">
         <div class="hero-section">
           <div class="hero-welcome-msg">
-            <h1 class="intro-msg">I’m</h1>
+            <h2 class="intro-msg">I’m</h2>
```

Comment:
- [ ] You should keep this title with `<h1>` cause it's the main title so please try change it back to h1

Reply:
I have implemented this changes